src/error-handler.ts

```diff
@@ -0,0 +1,77 @@
+import { isPlainObject } from 'lodash'
+import { AxiosError } from 'axios'
+
+/**
+ * Handles errors received from the server. Parses the error into a more useful
+ * format, places it in an exception and throws it.
+ * See https://www.contentful.com/developers/docs/references/errors/
+ * for more details on the data received on the errorResponse.data property
+ * and the expected error codes.
+ * @private
+ */
+export default function errorHandler(errorResponse: AxiosError): never {
+  const { config, response } = errorResponse
+  let errorName
+
+  // Obscure the Management token
+  if (config && config.headers && config.headers['Authorization']) {
```

Comment:
Should we also include the check for older node versions?
https://github.com/contentful/contentful.js/blob/5ae9e093f8541f3e6bad8c357ef88229adb85ab0/lib/contentful.js#L106-L108

Reply:
We don't need to here. What's deprecated is [_headers](https://nodejs.org/api/deprecations.html#DEP0066)

Reply:
That's what I meant. should we also try to obfuscate the token for older node versions (before depreciation)?

Reply:
The token is going to be obscured for older node versions here too. The difference between this and what we had in contentful.js is in addition to this config header, we had other places in the response with tokens and so we had to for each of those places obscure the token.
Now that we don't send the entire Axios error, we just need to do this once in this config header.

Reply:
Ah right, understood!